code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -413,16 +469,17 @@
 			to_chat(X, span_xenowarning("Why would we target ourselves?"))
 		return FALSE
 
-	if(!line_of_sight(X, A)) //Need line of sight.
-		if(!silent)
-			to_chat(X, span_xenowarning("We require line of sight to mark them!"))
-		return FALSE
+	if(require_los)
+		if(!line_of_sight(X, A)) //Need line of sight.
+			if(!silent)
+				to_chat(X, span_xenowarning("We require line of sight to mark them!"))
+			return FALSE
```

Comment:
as above, this is not used on the base ability type, so why is it even here?

Reply:
Well thought they could be used for future strains etc as needed.

Reply:
it's made to be usable in the future for others and so on easier so it makes sense this is here, also makes things easier in general.

Reply:
If its needed at the parent level in the future, than futurecoder can move it, otherwise it's just pointless bloat.